guides/client/js-interop.md

```diff
@@ -316,3 +326,29 @@ Hooks.Chart = {
 ```
 
 *Note*: In case a LiveView pushes events and renders content, `handleEvent` callbacks are invoked after the page is updated. Therefore, if the LiveView redirects at the same time it pushes events, callbacks won't be invoked on the old page's elements. Callbacks would be invoked on the redirected page's newly mounted hook elements.
+
+## JS commands
+
+*Note*: If possible, construct commands via Elixir using `Phoenix.LiveView.JS` and trigger them via Phoenix DOM [Bindings](bindings.md).
+
+While `Phoenix.LiveView.JS` allows you to construct a declarative representation of a command, it may not cover all use cases.
+In addition, you can execute commands that integrate with server DOM patching via JavaScript using:
+- Client hooks: `this.js()` or the
+- LiveSocket instance: `liveSocket.js()`.
+
+The command interface returned by `js()` above offers the following functions:
```

Comment:
Mostly duplicates `js_commands.js` JSDoc.

Remove one or the other, or accept duplication?

Reply:
Duplication is fine for now :)